src/main/java/com/codurance/retropolis/controllers/BoardController.java

```diff
@@ -0,0 +1,26 @@
+package com.codurance.retropolis.controllers;
+
+
+import com.codurance.retropolis.models.Board;
+import com.codurance.retropolis.services.BoardService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/board")
+public class BoardController {
+
+    private final BoardService boardService;
+
+    @Autowired
+    public BoardController(BoardService boardService) {
+        this.boardService = boardService;
+    }
+
+    @GetMapping
+    public Board getBoard() {
+        return boardService.getBoard();
```

Comment:
Great this is the new controller. I quite like the naming: getBoard() for the method.